### ISSUE TITLE
I-ALiRT: APL Meeting for Tori

### ISSUE THREAD

**Issue by laspsandoval** [IMAP-Science-Operations-Center/imap_processing#1111]
Attend the APL Meeting for I-ALiRT.

**Comment by bourque** [comment]
Just tidying up the project board. @laspsandoval @torimarbois could this be closed?

**Comment by torimarbois** [comment]
> Just tidying up the project board. [@laspsandoval](https://github.com/laspsandoval) [@torimarbois](https://github.com/torimarbois) could this be closed?

@bourque Yes! Weird that it still says Open even though the status is 'Done'.
